### DISCUSSION TITLE
MRTK gaze(head cursor) and mouse cosor comflict with eath other

### DISCUSSION THREAD

**Discussion by quater782** [Rufus31415/Simple-WebXR-Unity#59] [Q&A] [answered]
I am trying to mirgate my hl2 project to webGL. But there seems to be a conflict between head and mouse curosr. for example, if the two cursors are targeting two different buttons, then both button will be activated ."it active seletct at the same time". It will first exucte the event from button the mouse cursor selected, then the event from the head cursor. 
Anyone has any idea how can it be fixed : ( 
(btw the dedug experience is like hell.... omg)
![2809e85ebfcad75b928c244ef08440b](https://github.com/user-attachments/assets/f4a8e99b-30a1-49a1-b9c5-b5270b107db0)

**Comment by quater782** [comment, marked as answer]
...... debug makes a man dummy.  It is almost fixed by myself now

**Comment by Rufus31415** [comment]
I'm glad you found a way to fix this issue !
